locales/hi/php-inline.json

```diff
@@ -21,7 +21,7 @@
     "date_equals": "यह :date के बराबर तारीख होनी चाहिए ।",
     "date_format": "यह प्रारूप :format से मेल नहीं खाता है ।",
     "declined": "This value must be declined.",
-    "declined_if": "This value must be declined when :other is :value.",
+    "declined_if": ":Attribute को ज़रूर अस्वीकार किया जाना चाहिए जब :other :value हो ।",
```

Comment:
In the inline translation file, you do not need to access the attribute.

For example:

> Main: The :attribute must be declined when :other is :value.
> Inline: This field must be declined when :other is :value.